view/src/components/produce.js

```diff
@@ -0,0 +1,724 @@
+                          <Typography variant="body2" component="p">
+                            USDA Price: ${produce.price}
+                            <br />
+                            Average Price Paid by Farmlink: ${produce.pricePaid}
+                          </Typography>
+                        </Box>
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Internal Statistics:
+                          </Typography>
+                          <Typography variant="body2" component="p">
+                            Amount Moved by FarmLink (lbs):
+                            {produce.amountMoved}
+                          </Typography>
+                        </Box>
+                      </Box>
+                    </CardContent>
+                    <CardActions>
+                      <Button
+                        size="small"
+                        color="primary"
+                        onClick={() => this.handleViewOpen({ produce })}
+                      >
+                        View
+                      </Button>
+                      <Button
+                        size="small"
+                        color="primary"
+                        onClick={() => this.handleEditClick({ produce })}
+                      >
+                        Edit
+                      </Button>
+                      <Button
+                        size="small"
+                        color="primary"
+                        onClick={() => this.handleDelete({ produce })}
+                      >
+                        Delete
+                      </Button>
+                    </CardActions>
+                  </Card>
+                </Grid>
+              ))}
+            </Grid>
+          </Container>
+
+          <Dialog
+            onClose={handleViewClose}
+            aria-labelledby="customized-dialog-title"
+            open={viewOpen}
+            fullWidth
+            classes={{ paperFullWidth: classes.dialogStyle }}
+          >
+            <DialogTitle id="customized-dialog-title" onClose={handleViewClose}>
+              {this.state.name}
+            </DialogTitle>
+            <DialogContent dividers>
+              <Box
+                display="flex"
+                flexDirection="row"
+                flexWrap="wrap"
+                p={0}
+                m={0}
+              >
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Shipping Temperatures in Reefer (°F):
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Maintenance Temperature:
+                    {this.state.shippingMaintenanceTemperatureLow} -
+                    {this.state.shippingMaintenanceTemperatureLow}
```

Comment:
Should one of these be shippingMaintenanceTemperatureHigh?

Reply:
Good catch! This would have been very confusing